feat(content): New "obsolete" ships: Monitor & Jackal + Heavy Blaster (#11358)
endless-sky/endless-sky: data/human/weapons.txt

		"hull damage" 8.8
	description "This is a turreted version of the Modified Blaster, which is an Energy Blaster modified for greater power at the cost of higher energy consumption and heat. These turrets are popular with pirates and others who are trying to cram as much firepower into their ships as possible."

outfit "Heavy Blaster"
	category "Guns"
	series "Guns"
	index 01050

> 	cost 140000

Reviewer: ```suggestion
	cost 92000
```
140000 is a bit expensive for a blaster, 92k puts it just above a twin mod blaster.

Reviewer 2: This is quite a bit bigger than a twin mod blaster, though (19t vs 14t). It's almost as close to the plasma cannon (25t, 190k cost).

Reviewer: I was mostly comparing it to the regular blaster, thinking of it as a scaled up version, it does ~4x the damage in ~4x the space, so a pricetag close to 6x seems pretty reasonable. Additionally, 140k seems too close to the proton gun's price, which is a way more powerful weapon.

Reviewer 2: 114k would give it the same cost per ton as the twin modified blaster. Given that cost per ton seems generally to go up at least a little with size, you could maybe put it in the 120k-130k range?
```suggestion
	cost 125000
```

	thumbnail "outfit/heavy blaster"
	"mass" 19
	"outfit space" -19
	"weapon capacity" -19
	"gun ports" -1
	weapon
		sprite "projectile/heavy blaster"
		sound "heavy blaster"
		"hit effect" "blaster impact"
		"inaccuracy" 5
		"velocity" 9
		"lifetime" 65
		"reload" 30
		"firing energy" 49
		"firing heat" 150
		"shield damage" 90

> 		"hull damage" 54

Reviewer 3: I'm not a balance reviewer, but this seems slightly overpowered to me. Compared to the Modified Blaster, it has 2.5x the reload time, but does 6.8x the shield damage and 6.1x the hull damage. The bolts also travel for longer, even if they're slightly slower. Is there any reason it has improved so much?
Holding off on a description until this is clarified.

		"hull damage" 8.8
	description "This is a turreted version of the Modified Blaster, which is an Energy Blaster modified for greater power at the cost of higher energy consumption and heat. These turrets are popular with pirates and others who are trying to cram as much firepower into their ships as possible."

outfit "Heavy Blaster"
	category "Guns"
	series "Guns"
	index 01050
	cost 125000
	thumbnail "outfit/heavy blaster"
	"mass" 19
	"outfit space" -19
	"weapon capacity" -19
	"gun ports" -1
	weapon
		sprite "projectile/heavy blaster"
		sound "heavy blaster"
		"hit effect" "blaster impact"
		"inaccuracy" 5
		"velocity" 9
		"lifetime" 65
		"reload" 30
		"firing energy" 49
		"firing heat" 150
		"shield damage" 90
		"hull damage" 54

> 	description "The Heavy Blaster is a scaled-up variant of the ever-so-popular Energy Blaster design. These saw extensive use on several warships earlier in history, most notably as the main armament of the Firebird when it was first produced. Over time, the development of more advanced weapons such as the Particle Cannon and Plasma Cannon phased out usage of the Heavy Blaster, but it still performs consistently enough to be found on older serviceable ships."

Reviewer 4: I think of organizations as being responsible for "phasing out" things, rather than the process of development itself. Maybe this works better?
```suggestion
	description "The Heavy Blaster is a scaled-up variant of the ever-so-popular Energy Blaster design. These saw extensive use on several warships earlier in history, most notably as the main armament of the Firebird when it was first produced. Over time, the development of more advanced weapons such as the Particle Cannon and Plasma Cannon has supplanted the Heavy Blaster on newer craft, but it still performs consistently enough to be found on older serviceable ships."
```

Reviewer 3: You'd need to use "have" instead of "has" there, but that's definitely better otherwise.

Reviewer 4: No, I don't think so. "The development" (of multiple things, admittedly) is what's doing the supplanting here, and it's singular.

		"firing heat" 150
		"shield damage" 90
		"hull damage" 54

> 	description "The Heavy Blaster is a scaled-up variant of the ever-so-popular Energy Blaster design. These would see extensive use on several warships earlier in history, most notably as the main armament of the Firebird when it was first produced. Over time, the development of more advanced weapons such as the Particle or Plasma Cannon phased out the usage of the Heavy Blaster, however, it still performs consistently enough to be found on older ships still in service."

Reviewer 3: ```suggestion
	description "The Heavy Blaster is a scaled-up variant of the ever-so-popular Energy Blaster design. These saw extensive use on several warships earlier in history, most notably as the main armament of the Firebird when it was first produced. Over time, the development of more advanced weapons such as the Particle Cannon and Plasma Cannon phased out usage of the Heavy Blaster, but it still performs consistently enough to be found on older serviceable ships."
```
